recipe/meta.yaml

```
@@ -30,7 +30,7 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
   host:
-    - python
+    - python >=3.8
```

```suggestion
    - python
```
The Python version constraint cannot be listed here for non-noarch packaged and instead is controlled by the build section; see line 18.